testutil-server/src/main/java/io/spine/testing/server/blackbox/BlackBoxBoundedContext.java

```diff
@@ -105,26 +106,47 @@
     private static final TransportFactory transportFactory = InMemoryTransportFactory.newInstance();
 
     private final BoundedContext boundedContext;
+
+    /**
+     * Collects all commands, including posted to the context during its setup or
+     * generated by the entities in response to posted or generated events or commands.
+     */
     private final CommandCollector commands;
+
+    /**
+     * Commands received by this instance and posted to Command Bus during the test setup.
+     *
+     * <p>These commands are filtered out from those which are generated in the Bounded Context
```

Comment:
Let's remove `which are` from this line to avoid doubling the "which".
```
These commands are filtered out from those generated in the Bounded Context...
```